build/circle-trigger-nightly.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -eu
+
+REV=$(git describe --tags)
```

Comment:
But this is surprising since it means that the cron job should not be run from a normal working copy of the repo; it needs its own clone (and something, presumably another cron job, will be responsible for updating it). And I think we *do* want to special-case master: if this script is for some reason run from a branch other than master, we don't want it to create issues.